Lib/tkinter/test/test_ttk/test_extensions.py

```diff
@@ -291,6 +291,32 @@ def cb_test(item):
 
         optmenu.destroy()
 
+    def test_unique_radiobuttons(self):
+        # check that radiobuttons are unique across instances (bpo25684)
+        items = ('a', 'b', 'c')
+        default = 'a'
+        optmenu = ttk.OptionMenu(self.root, self.textvar, default, *items)
+        textvar2 = tkinter.StringVar(self.root)
+        optmenu2 = ttk.OptionMenu(self.root, textvar2, default, *items)
+        optmenu.pack()
+        optmenu.wait_visibility()
+        optmenu2.pack()
+        optmenu2.wait_visibility()
+        optmenu['menu'].invoke(1)
+        optmenu2['menu'].invoke(2)
+        optmenu_radiobutton_name = optmenu['menu'].entrycget(0, 'variable')
+        optmenu2_radiobutton_name = optmenu2['menu'].entrycget(0, 'variable')
+        self.assertNotEqual(optmenu_radiobutton_name,
+                            optmenu2_radiobutton_name)
+        self.assertEqual(self.root.tk.globalgetvar(optmenu_radiobutton_name),
+                         items[1])
+        self.assertEqual(self.root.tk.globalgetvar(optmenu2_radiobutton_name),
+                         items[2])
+
+        optmenu.destroy()
+        del textvar2, optmenu_radiobutton_name, optmenu2_radiobutton_name
```

Comment:
Thanks.  For some reason, even though they were local, I thought that the Tk variables had to be explicitly deleted for refleaks.  But, it's probably just when it's in setUp.